packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -28,6 +28,105 @@ Email templates use common and unique [Expression Language (EL) variables](https
 
 > **Note:** Some templates listed in the [variables tables](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-expression-language) may not appear in your org. To obtain these templates, contact [Okta Support](https://support.okta.com/help/s/?_ga=2.17747641.1660906902.1597076228-1076744453.1575496867).
 
+### Customize your org's templates by using Velocity Templating Language (VTL)
+<ApiLifecycle access="ea" />
+
+#### Overview of what this EA feature enables
+
+This Early Access (EA) feature provides customers with the functionality to customize the email templates, which Okta provides, that allows them to send email notifications to their end users. It changes the underlying email templating engine to Velocity so that you can use the syntax of the Velocity Templating Language (VTL) to customize your org's email templates. In addition to the new syntax, which provides enhanced conditional logic, some org attributes and access are provided to all attributes in the Okta [User Profile object](/docs/reference/api/users/#profile-object).
+
+##### What this EA feature doesn't enable
+
+Okta will not perform any automated migration of previously customized email templates for EA. This means that when you enable the feature flag to access the functionality, all email templates will revert to defaults.
+
+However, when you disable the feature flag, your old customized templates will return. If you choose to disable the feature flag, you will lose access to the customizations done while the EA was enabled.
+
+##### EA Feature requirements
+
+To access this EA feature, you need the following prerequisites:
+
+* Requires an Okta preview org (`example.oktapreview.com`) and Okta production org (`example.okta.com`)
+* Requires the self-service feature flag, `ENHANCED_EMAIL_MACROS`, to be enabled for the org
+
```

Comment:
Also to edit the email macros, you need to disable `FREE_TRIAL_PHISHING_WARNING`

Reply:
We shouldn't doc that FF, but you could say "For free orgs, please contact Okta support (support@okta.com) to request email customization" 

And I noticed we even have a banner here that says free trial editions cannot customize emails: https://developer.okta.com/docs/guides/email-customization/customize-email-templates/ 

So we should just make things consistent, I think we allow free orgs to customize they just have to contact support

Reply:
Fixed